### PR TITLE
Create a test mod for the fabric-content-registries-v0 module

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

**Review by Technici4n — APPROVED** [review]
LGTM, does it test every feature of the module?

**Comment by Juuxel** [comment]
I think testing tag-based fuel values is the only thing that's missing from here.

**Comment by Technici4n** [comment]
Might be worth adding that, then?

**Comment by haykam821** [comment]
It does not test `LootEntryTypeRegistry` which is deprecated in favor of a replacement from another module as well. Testing tags is a good idea, though, and would be beneficial for #1476.

**Review by Juuxel — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
On a side note, I hope we have a place where we look up all the desired behaviors as a result of enabling these test mods than having to look into individual code. Another approach may be to have some unit tests in events, but those would be a bit hard to regulate as well.

File: fabric-content-registries-v0/src/testmod/java/net/fabricmc/fabric/test/registry/ContentRegistryTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public final class ContentRegistryTest implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		CompostingChanceRegistry.INSTANCE.add(Items.OBSIDIAN, 0.5F);
+
+		FlammableBlockRegistry.getDefaultInstance().add(Blocks.OBSIDIAN, 2, 3);
+		FlammableBlockRegistry.getDefaultInstance().add(BlockTags.SNOW, 4, 6);
+
+		FuelRegistry.INSTANCE.add(Items.OBSIDIAN, 20 * 3);
+		FuelRegistry.INSTANCE.add(ItemTags.FOX_FOOD, 20 * 6);
```
Comment (modmuss50) [inline fabric-content-registries-v0/src/testmod/java/net/fabricmc/fabric/test/registry/ContentRegistryTest.java:38]: Can we add a check on first tick or something to ensure these have been registered, and crash hard if they havent.

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Would perfer some validation that this has actually worked, crashing the game hard if something isnt quite right would be best.

**Comment by Technici4n** [comment]
@haykam821 Any update?

**Comment by Technici4n** [comment]
up?

**Comment by ghost** [comment]
Doesn't this already exist? In fact there are conflicts...

**Comment by ghost** [comment]
Note that Composting and Fuel tests are missing from testmods from all the releases (from 1.17 to 1.19).
All other tests are present.

**Comment by Technici4n** [comment]
Feel free to open a new PR with proper automated testing then! :)

**Pull request closed without merging**